### ISSUE TITLE
"Syntax error" during expectation creation when adding binary body to ResponseBuilder

### ISSUE THREAD

**Issue by amkoro** [mcustiel/phiremock#30]
I found the issue during expectation creation when I tried to add a binary data from image:
```
$expectation = Phiremock::on(
    A::getRequest()
        ->andUrl(Is::equalTo('/test_images/image1.jpg'))
)->then(
    Respond::withStatusCode(200)
        ->andBody(file_get_contents($testDataPath.'test_images/image1.jpg'))
        ->andDelayInMillis(2000)
);
```

Package versions:
```
mcustiel/creature                        v2.0.0
mcustiel/mockable-datetime               1.0.0
mcustiel/phiremock                       v1.7.1
mcustiel/phiremock-codeception-extension dev-master e52996c
mcustiel/php-simple-di                   1.2.1
mcustiel/php-simple-request              v3.1.0
mcustiel/power-route                     v3.0.0
```

Error:
```
An error occurred creating the expectation: Syntax error
.../vendor/mcustiel/phiremock/src/Client/Phiremock.php:281
.../vendor/mcustiel/phiremock/src/Client/Phiremock.php:269
.../vendor/mcustiel/phiremock/src/Client/Phiremock.php:74
.../vendor/mcustiel/phiremock-codeception-extension/src/Module/Phiremock.php:59
```

**Comment by mcustiel** [comment]
Hi @amkoro 
Thanks for reporting it, I will give it a look as soon as I have some time.

**Comment by mcustiel** [comment]
Fixed in version v1.7.2: https://github.com/mcustiel/phiremock/releases/tag/v1.7.2
